### ISSUE TITLE
Numpy, np.float issue when I try pythonw __main__.py

### ISSUE THREAD

**Issue by YashIsACoder** [andycasey/smhr#335]
![Image 6-8-2023 at 10 12 am](https://github.com/andycasey/smhr/assets/135817902/d33f639c-e5f3-49bb-b39b-ddb807ab8d93)

Hi, when I try to run pythonw __main__.py, I keep on getting this error and I'm not sure how to resolve this issue. For reference, I am using a Macbook Pro with a M1 chip. I have tried doing 'python __main__.py' instead and I get the same issue, as well as deactivating and reactivating my anaconda environment or reinstalling smhr. What should I do next?

Thank you in advance.

**Comment by alexji** [comment]
This is a known error due to an update to numpy. The solution is to replace
np.float with float, which I will get to when I have a chance in a few
weeks. For now you can modify the code yourself (and then submit a pull
request!), or you can downgrade your version of numpy (something like 1.10
or 1.8 should work).

**Comment by YashIsACoder** [comment]
Hi, this issue in particular does not appear anymore however I am getting a similar issue when trying to run smhr
  
![image](https://github.com/andycasey/smhr/assets/135817902/570989f9-6ac9-44db-82fd-b64967aabe75)

For reference, I replaced all the np.float code in the 4 scripts with float. From what I see, I think I need to replace np.int with int but could I get clarification if I should instead replace it with np.int64.

Thank you once again.

**Comment by andycasey** [comment]
int will be fine

**Comment by samantha-usman** [comment]
Making a note that I ran into this problem as well. I tried manually replacing `np.float` with `float` (and `np.int` with `int`) in the offending file (`smhr/smh/radiative_transfer/moog/cog.py`), but it still ran the same error, specifically at line 232 even though I now have `dtype=float` instead of `dtype=np.float`.

I couldn't figure out why it was still breaking since I'd replaced all the `np.float` with `float`, so I copped out and just reverted to an older version of `numpy` via `conda install numpy < 1.23` and it worked fine.
